### ISSUE TITLE
Update UI for R2 scores

### ISSUE THREAD

**Issue by hjwilli** [EpistasisLab/pennai#315]
The metric used for comparing regression experiments is the [R2 score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html).  The max value is 1.0, and it can be negative.  Update the UI for R2 scores on regression experiments.

* change labels from 'R2' -> 'R^2'
* update range of best result bar on datasets page to be a range, not a percentage

**Comment by lacava** [comment]
We could use [Pearson's R](https://github.com/EpistasisLab/pennai/blob/4b237de77ea267492e9102072d5e273cd2989162/machine/learn/skl_utils.py#L116)? That would give a value between -1 and 1.  Squaring it would give Pearson's R^2 between 0 and 1.

**Comment by hjwilli** [comment]
addressed with https://github.com/EpistasisLab/pennai/pull/316, closing.
